docs/4.3. Behaviour - Tokens.md

```diff
@@ -97,28 +107,23 @@ access to any of the APIs. A missing API key MUST indicate no access to that spe
 }
 ```
 
-In addition, a set of universal `key:value` pairs MAY be included for each API, with the `keys` indicating the type
-of access to the API and the `value` indicating individual paths that are permitted to be accessed in such a way.
-Wild-carding is permitted in the path specifiers. Individual AMWA specifications MAY also specify additional
-key:value pairs specific to the API. General access keys include:
-*   **create** - ability to insert a new resource. This usually takes the form of POST HTTP request verbs.
+The `value` for each NMOS API indicates the access rights of the given user for that specific API. The value required
+by the Resource Server MUST exactly match one of the values in the array. General access rights include:
+*   **write** - ability to insert a new, or edit an existing, resource. This usually takes the form of POST, PUT or
```

Comment:
We need to be clearer here rather than saying 'usually'. I'd suggest listing the full set of potential HTTP methods either under write or read and saying that's exactly what they mean.